scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/JobResultView.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
```

Comment:
You should consider updating your java class template.

Reply:
You are right. 
Bad news. We have common template for all projects. 
So somebody have to update all projects.